microsoft-edge/devtools-guide-chromium.md

```diff
@@ -12,104 +12,140 @@ keywords: microsoft edge, web development, f12 tools, devtools
+Microsoft Edge has adopted the Chromium open source project to create better web compatibility and less fragmentation of different underlying web platforms.  This change should make it easier for you to build and test your websites in Microsoft Edge and ensure that each works as expected even while your users are browsing in a different Chromium-based browser \(such as Google Chrome, Vivaldi, Opera, or Brave\).  
 
-As the web has grown in usage across an ever-widening array of device types, the complexity and overhead involved in testing web sites has exploded. Since web developers— particularly those at small companies— need to test so many different systems, it's nearly impossible to ensure that interesting sites will work well across all device types and all browsers. With the next version of Microsoft Edge, we hope to simplify this matrix for you by aligning the Microsoft Edge web platform with other Chromium-based browsers and providing a best-in-class developer tooling experience, both inside the browser and with the other developer tools you use every day, like Visual Studio Code.
+As the web has grown in usage across an ever-widening array of device types, the complexity and overhead involved in testing websites has exploded. Since web developers \(particularly those at small companies\) must test against so many different systems, you may find it nearly impossible to ensure that sites work well on all device types and all browsers.  Now that Microsoft Edge is based on Chromium, the Microsoft Edge team has simplified the matrix by aligning the Microsoft Edge web platform with other Chromium-based browsers and provided a best-in-class developer tooling experience, both inside the browser and with the other developer tools you use every day \(such as Visual Studio Code\).  
 
-If you are checking out Microsoft Edge and you mainly develop in a Chromium-based browser, you should feel right at home.  The Microsoft Edge \(Chromium\) Developer Tools are exactly like the developer tools you already know and use.  So far, we have made a couple changes we hope you love so for more info, check out [What's new in the Microsoft Edge \(Chromium\) DevTools](./devtools-guide-chromium/whats-new.md).  
+If you are checking out Microsoft Edge and you mainly develop in a Chromium-based browser, you should feel right at home.  The Microsoft Edge \(Chromium\) Developer Tools function in the same way as the developer tools you already know and use.  For more info, see [What's new in the Microsoft Edge (Chromium) DevTools][DevtoolsChromiumWhatsNew].  
```

Comment:
@jm-trd-ms, is there a way we could import a bit of Markdown that always points to the latest version of What's New?We wrote this What's New doc over a year ago and it's actually not even represented in the toc.yml any more. It would make sense to me to have one place we can go when What's New is updated and then just have that imported here. Is that possible? How do we set that up?

Reply:
Related: https://github.com/MicrosoftDocs/edge-developer/issues/639

Reply:
Tracking this a separate issue per Rachel's comment.